index.d.ts

```diff
@@ -8,19 +8,20 @@ export declare type BBox = [number, number, number, number];
 export declare type Google = [number, number, number];
 export declare type Tile = [number, number, number];
 export declare type Quadkey = string;
+export declare type Accurancy = { enable: Boolean, decimal, Number};
```

Comment:
remove `enable` from `Accuracy`

By simply providing `decimal` should be enough to define the accuracy type